src/pydata_sphinx_theme/bootstrap_html_translator.py

```diff
@@ -1,6 +1,6 @@
 """A custom Sphinx HTML Translator for Bootstrap layout
 """
-from distutils.version import LooseVersion
+from packaging.version import Version
```

Comment:
Seems like the new way to do versions comparisons. I would suggest to add `packaging` in the deps. The reason it's working is because `sphinx` has it in his deps, but using this here would make it a direct deps for this package as well.

Reply:
Thanks for checking! I will add this as a dependency.